plugins/mcreator-localization/help/default/particle/rotation_provider.md

```diff
@@ -0,0 +1,7 @@
+This parameter controls if the particle billboards (always appears to face player)

```

Comment:
> This parameter controls if the particle billboards (always appears to face player)

IIRC particles billboard by default, so it would be beter to say "this parameter overrides default billboarding (always appears to face player) behavior" or something like that